src/ice/pair.rs

```diff
@@ -2,16 +2,12 @@ use std::collections::VecDeque;
 use std::fmt;
 use std::time::{Duration, Instant};
 
-use crate::io::{stun_resend_delay, STUN_MAX_RETRANS};
-use crate::io::{Id, TransId, STUN_MAX_RTO_MILLIS};
+use crate::io::{Id, StunTiming, TransId};
 use crate::Candidate;
 
-const MIN_TIMEOUT: Duration = Duration::from_millis(STUN_MAX_RTO_MILLIS);
-
 // When running ice-lite we need a cutoff when we consider the remote definitely gone.
 const RECENT_BINDING_REQUEST: Duration = Duration::from_secs(15);
 
-#[derive(Default)]
```

Comment:
What's the thinking here? Better to be explicit? Considering it adds all those lines below.

Reply:
The thinking was that setting `StunTimings` to default is semantically wrong because it needs to be copied over and set and not initialised.

Reply:
Right. Ok.